UI/xdg-data/com.obsproject.Studio.metainfo.xml.in

```diff
@@ -30,16 +32,32 @@
   <url type="bugtracker">https://github.com/obsproject/obs-studio/issues</url>
   <url type="donation">https://obsproject.com/contribute</url>
   <url type="translate">https://crowdin.com/project/obs-studio</url>
+
+  <recommends>
+    <internet>always</internet>
+
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </recommends>
+
+  <supports>
+    <internet>offline-only</internet>
+  </supports>
+
   <screenshots>
     <screenshot type="default">
       <image>https://obsproject.com/assets/images/OBSDemoApp2610.png</image>
+      <caption>The OBS Studio main window with grey rectangles, and SMPTE color bars</caption>
     </screenshot>
   </screenshots>
+
+  <branding>
+    <color type="primary">#284cb8</color>
+  </branding>
+
   <content_rating type="oars-1.1"/>
-  <kudos>
-    <kudo>ModernToolkit</kudo>
-    <kudo>HiDpiIcon</kudo>
-  </kudos>
+
   <releases>
     <release version="@OBS_VERSION@" date="@APPDATA_RELEASE_DATE@"/>
```

Comment:
Would this below the `<release>` element work?

```xml
<url type="details">https://github.com/obsproject/obs-studio/releases/tag/@OBS_VERSION@</url>
```

Reply:
Added!